src/app.js

```diff
@@ -150,8 +157,8 @@ gpii.app.updateMenu = function (tray, menuTemplate, events) {
   * Currently uses an url to key in although this should be changed to use Electron IPC.
   * @param token {String} The token to key in with.
   */
-gpii.app.keyIn = function (token) {
-    request("http://localhost:8081/user/" + token + "/login", function (/*error, response, body*/) {
+gpii.app.keyIn = function (that, token) {
```

Comment:
Is there a reason the signature changed here?

Reply:
It's probably a leftover from some of the initial drafts. I'll remove it right away.